covid19-dashboard/README.md

```diff
@@ -62,41 +55,102 @@ in the US with NYT COVID-19 data.
+        2. Only index divisible by clean_step_size will be kept.
+        3. If clean_step_size === 1, keep all values.
+        3. Example: clean_step_size = 2, ["Jan", "Feb", "Mar", "Apr", "May"] => ["Jan", "Mar", "May"] because indexes 0, 2 and 4 are divisible by 2.
+
+### calculate_data() performs the following calculations
 
     1. Moving Average (N-days).
         1. For every date in the input, sum up the values of the previous N-days and divide by N-days.
-    2. Per Capita of the Moving Average.
+        2. If Moving Average == 0, skip the moving average calculation. Use raw data instead.
+    2. Cumulative:
+        1. For every date in the input, sum up the values of all the previous days.
+    3. Per Capita of the Moving Average or Raw.
         1. For every date in the Moving Average output, divide by the place's total population.
-        2. Multiply by 1M to amplify data -- 0.00001 numbers are not too friendly).
-    3. Percent Change of the Moving Average.
+        2. Multiply by 1M to amplify data -- 0.00001 numbers are not too friendly.
+    4. Percent Change of the Moving Average or Raw.
         1. For every date in Moving Average, divide by the value from (date - date_from_baseline).
         2. Subtract 1 to the value.
         2. Multiply by 1M to convert to a percent.
-        
 
-Outputs the following:
-    1. A dictionary of type:
+## Front-End
+
+Depending on whether the user visits `/dashboard/covid19` or `/dashboard/socialWellness`, a different configuration file will be used to draw the table.
+The browser will use `src/covid19.json` for the COVID-19 dashboard.
+The browser will use `src/sociallWelness.json` for the Sociall Wellness dashboard.
+The configuration files have a `table` property with a list of category objects.
 
+### Example of Category Object
+
+```
     {
-        "movingAverage": {date: value},
-        "perCapita": {date: value},
-        "pctChange": {date: value}
+        # title of the category.
+        "title": "Above Poverty",
+        # key of the category to retrieve data from API. Data must exist in API response, or will be skipped.
+        "id": "cumulativeAbovePovertyInLast12Months",
+        # what type of graph to draw for this category. View ./src/Th.tsx to add or remove any types.
+        "typeOf": "lineGraph",
+        # The color of the graph. Any CSS color will also work. View CSS file for any predefined colors.
+        "color": "--dc-red-strong",
+        # when the user hovers on the graph, the subtitle of the graph.
+        "graphSubtitle": "Total Population Above Poverty (Last 12 Months)",
+        # is the category enabled by default on page load.
+        "enabled": false
     }
-   
\ No newline at end of file
+```
+
+## Local Development
+
+### Set Google Application Credential
+
+**NOTE:** This project is currently deployed under the GCP Project `datcom-wesbite`, but any other project can be used instead.
+
+```bash
+export GOOGLE_CLOUD_PROJECT="datcom-website"
```

Comment:
include this in the script for those of us who jump around multiple projects?
see the run_server.sh script in the website repo for an example

Reply:
done